code_to_gast/routers/py_router.py

```diff
@@ -51,4 +56,5 @@ def node_to_gast(node):
 
 
     else:
+        print(astor.dump(node))
```

Comment:
Just making sure this is intentional however, I do think it would make sense to print out these error cases so we can pull them from logs later so I say keep it 👍